apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -237,16 +243,35 @@ export default class ChangeAction extends BaseRushAction {
               type: bumpType
             } as IChangeInfo;
           });
-        } else {
-          return {
-            comment: '',
-            packageName: packageName,
-            type: 'none'
-          } as IChangeInfo;
         }
       });
   }
 
+  private _getBumpOptions(packageName: string): {[type: string]: string } {
+    const project: RushConfigurationProject | undefined = this.rushConfiguration.getProjectByName(packageName);
+    const versionPolicy: VersionPolicy | undefined = project!.versionPolicy;
+
+    let bumpOptions: { [type: string]: string } = {
```

Comment:
>bump [](start = 8, length = 4)

"bump" is not professional terminology.

Could we call it something like "increment" or "upgrade"?

Reply:
It is used everywhere in semver.

Reply:
It is used everywhere in semver.

---
In reply to: [147508119](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147508119) [](ancestors = 147508119)

Reply:
Example?

---
In reply to: [147509693](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147509693) [](ancestors = 147509693,147508119)

Reply:
[http://semver.org/](http://semver.org/) 

---
In reply to: [147509852](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147509852) [](ancestors = 147509852,147509693,147508119)

Reply:
I only see one mention of it in a FAQ answer

---
In reply to: [147510469](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147510469) [](ancestors = 147510469,147509852,147509693,147508119)

Reply:
[https://docs.npmjs.com/cli/version](https://docs.npmjs.com/cli/version) 

---
In reply to: [147510748](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147510748) [](ancestors = 147510748,147510469,147509852,147509693,147508119)